spec/support/corpus.rb

```diff
@@ -134,6 +134,7 @@ def install_mutant
         repo_path.join('Gemfile').open('a') do |file|
           file << "gem 'mutant', path: '#{relative}'\n"
           file << "gem 'mutant-rspec', path: '#{relative}'\n"
+          file << "eval_gemfile File.expand_path('#{relative.join('Gemfile.shared')}')\n"
```

Comment:
Please see my comment about the intentions of `Gemfile.shared` here: https://github.com/mbj/mutant/pull/439#discussion_r38951885